Source/WebCore/platform/network/HTTPParsers.cpp

```diff
@@ -976,11 +958,14 @@ bool isCrossOriginSafeRequestHeader(HTTPHeaderName name, const String& value)
         break;
     }
     case HTTPHeaderName::Range:
-        if (!isSimpleRangeHeaderValue(value))
+        long long start;
+        long long end;
+        if (!parseRange(value, false, start, end))
```

Comment:
It is not easy to understand what false means here.
We tend to prefer boolean enums so that it would read like:
`parseRange(value, AllowWhitespace::Yes, start, end)`